recipes/songbird/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "0.8.2" %}
+
+package:
+  name: songbird
+  version: {{ version }}
+
+source:
+  fn: songbird-{{ version }}.tar.gz
```

Comment:
You can drop the filename; it is no longer needed.

Reply:
👍